generators/client/templates/angular/src/main/webapp/app/shared/language/_translate-partial-loader.provider.ts

```diff
@@ -31,8 +31,12 @@ export class TranslatePartialLoader implements TranslateLoader {
         return Observable.create(observer => {
             this.http.get(`${this.prefix}/${lang}/${part}${this.suffix}`).subscribe((res) => {
                 let responseObj = res.json();
-                Object.keys(responseObj).forEach(key => {
-                    combinedObject[key] = responseObj[key];
+                Object.keys(responseObj).forEach(key=>{
+                    if(!combinedObject[key]) {
+                        combinedObject[key] = responseObj[key];
+                    }else{
+                        $.extend(true,combinedObject[key], combinedObject[key], responseObj[key])
```

Comment:
The issue was when we were loading multiple part files with the same key, it would overwrite the objet and some fields where not translated (for example in entities when you chose enum fields  you had 2 différent file with the same key one for the entitity itself and one for the enum)